Browser_IDE/stylesheet.css

```diff
@@ -472,4 +541,73 @@ li.CodeMirror-hint-active {
   position: relative;
   flex: 1 1 auto;
   padding: var(--bs-modal-padding);
-}
\ No newline at end of file
+}
+
+/* Modal UI changes */
+
+#confirmationModal .sk-header, #projectConflictModal .sk-header {
```

Comment:
These modal changes are good, but right now they're hardcoded to only work on the `projectConflictModal` and `confirmationModal` modals - make sure they'll work on all modals.